test/functional/wallet_miniscript.py

```diff
@@ -266,6 +286,7 @@ def run_test(self):
                 ms["locktime"],
                 ms["sigs_count"],
                 ms["stack_size"],
+                ms.get("sha256_preimages"),
```

Comment:
comment above should be updated I think:

"# Test we can sign most Miniscript (all but ones requiring preimages, for now)"

Reply:
Thanks, updated.